### PR TITLE
added possibility to subselect analyses #60

### PR DESCRIPTION
**Pull request by LeonSaal**
Hi Rick,

I added the subselection as described in #60. I was also able to mitigate your fear of long assignments in https://github.com/rickhelmus/patRoon/issues/60#issuecomment-1348700988

I hope I didn't break anything - my tests were successful!

Kind regards

Leon

### CODE REVIEW

File: R/project-tool.R [hunk 1]
```diff
@@ -85,6 +85,12 @@ getScriptCode <- function(input, analyses)
         {
             addComment("Load analysis table", condition = comment)
             addCall(anaInfoVarName, "read.csv", list(value = anaTableFile, quote = TRUE))
+            if (any(anaTable$exclude))
```
Comment (rickhelmus) [inline R/project-tool.R:88]: I think it would be nicer if the actual csv doesn't contain the ignored files, i.e. by excluding it when calling `write.csv`.

**Comment by rickhelmus** [comment]
Excellent, many thanks!

Would you mind adding a line to `NEWS.md` reflecting your changes?

**Comment by LeonSaal** [comment]
Hi Rick,

I modified the 'table' option as suggested and updated `NEWS.md` accordingly.

Kind regards

Leon

**Comment by rickhelmus** [comment]
Thanks again! I will probably merge everything in tomorrow.

**Pull request merged**